railgun/pkg/config/config.go

```diff
@@ -94,6 +95,8 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	// Logging configurations
 	flagSet.StringVar(&c.LogLevel, "log-level", "info", `Level of logging for the controller. Allowed values are trace, debug, info, warn, error, fatal and panic.`)
 	flagSet.StringVar(&c.LogFormat, "log-format", "text", `Format of logs of the controller. Allowed values are text and json.`)
+	flagSet.BoolVar(&c.LogReduceRedundancy, "log-reduce-redundancy", false, `If enabled, repetitive log entries are suppressed. Built for testing environments - production use not recommended.`)
```

Comment:
how about prefix w/ test-log-reduce-redundancy since it is for test purposedly.

Reply:
Yes I agree. However would `debug` be a better indicator for the user?
```suggestion
	flagSet.BoolVar(&c.LogReduceRedundancy, "debug-log-reduce-redundancy", false, `If enabled, repetitive log entries are suppressed. Built for testing environments - production use not recommended.`)
```

Reply:
Fair enough. Implemented this change in 0e7761ef because it required changes in several places.